Sources/BraveWallet/Panels/RequestContainerView.swift

```diff
@@ -51,7 +51,10 @@ struct RequestContainerView<DismissContent: ToolbarContent>: View {
               cryptoStore: cryptoStore,
               keyringStore: keyringStore,
               networkStore: cryptoStore.networkStore,
-              onDismiss: onDismiss
+              onDismiss: {
+                cryptoStore.updateAssets()
```

Comment:
I don't think this is needed if we move update logic to `onNetworkListChanged()`, however it does make me realize we are also missing the logic to add as visible user asset when `AddSuggestedTokenView` adds a new asset 😮.

Reply:
updated in [3ae6668](https://github.com/brave/brave-ios/pull/7625/commits/3ae66680a0580698e1bdd31a74d0ce0faf705cb9)